apps/dashboard/app/views/support_ticket/email/email_template_override.text.erb

```diff
@@ -0,0 +1 @@
+support ticket email override
```

Comment:
This looks like it's just a test file. Can we move it to a `test/fixtures/config/views/support_ticket/email` directory and stub `Configuration.config_root`.

Reply:
I was trying to override the location mocking different things, but it didn't work.
I just realized that the test environment has been configured to use that folder for test views:

`config.paths["app/views"].unshift "test/fixtures/config/views"`

Updating....